packages/twenty-server/src/workspace/messaging/services/messaging-utils.service.ts

```diff
@@ -178,7 +180,7 @@ export class MessagingUtilsService {
         [participant.handle],
       );
 
-      const participantPersonId = participantPerson[0]?.id;
+      let participantPersonId = participantPerson[0]?.id;
 
       const workspaceMember = await manager.query(
```

Comment:
Makes me think we should move those raw queries to dedicated services now that we have them. This can be done later.

Reply:
Done